zproject/settings.py

```diff
@@ -490,12 +490,23 @@ def get_secret(key):
     # PRODUCTION.  Saves a bunch of time.
     PASSWORD_HASHERS = (
                 'django.contrib.auth.hashers.SHA1PasswordHasher',
-                'django.contrib.auth.hashers.PBKDF2PasswordHasher'
+                'django.contrib.auth.hashers.PBKDF2PasswordHasher',
+                'django.contrib.auth.hashers.Argon2PasswordHasher'
```

Comment:
I included argon here for checking because it is preferred in production as PBKDF2.